tools/mo/openvino/tools/mo/ops/split.py

```diff
@@ -78,8 +78,7 @@ def infer(node):
                         node['split_lengths'] = split_lengths
                     delete_out_port(i, node)
                 else:
-                    log.error("Zero dimension on {} branch after Split node {}".format(i, node.id))
-                    return
+                    log.warning("Zero dimension on {} branch after Split node {}".format(i, node.id))
```

Comment:
There is a TF model with RFFTxD and IRFFTxD operations and with the following property. 

The model has `Split` node with `0` in `split_lengths` input and the output port corresponding to the value `0` is connected with some input ports of other nodes.

For such model, the statement
```python3
                    log.error("Zero dimension on {} branch after Split node {}".format(i, node.id))
``` 
raises an exception, and the conversion is failed.

Because consumers of the above mentioned `Split` node are only `Concat` nodes, the conversion failure is incorrect behavior. 

We can replace `log.error` with `pass`, but I believe that we should give some diagnostic message instead of `pass`.